src/Cameras/VR/vrExperienceHelper.ts

```diff
@@ -316,10 +320,15 @@ export class VRExperienceHelper {
     private _onVRRequestPresentComplete: (success: boolean) => void;
 
     /**
-     * Observable raised when entering VR.
+     * Observable raised right before entering VR.
      */
     public onEnteringVRObservable = new Observable<VRExperienceHelper>();
 
+    /**
+     * Observable raised when entering VR has completed.
+     */
+    public onAfterEnteringVRObservable = new Observable<OnAfterEnteringVRObservableEvent>();

```

Comment:
why not just a boolean? (As the OnAfterEnteringVRObservableEvent is no exported)

Reply:
In case additional info is needed in the future, should this be exported in this case?

Reply:
Yep